demisto_sdk/commands/common/errors.py

```diff
@@ -299,9 +299,14 @@
     "invalid_generic_field_group_value": {'code': "GF100", 'ui_applicable': False, 'related_field': 'group'},
     "invalid_generic_field_id": {'code': "GF101", 'ui_applicable': False, 'related_field': 'id'},
     "non_default_additional_info": {'code': "IN142", 'ui_applicable': True, 'related_field': 'additionalinfo'},
-    "missing_default_additional_info": {'code': "IN143", 'ui_applicable': True, 'related_field': 'additionalinfo'}
+    "missing_default_additional_info": {'code': "IN143", 'ui_applicable': True, 'related_field': 'additionalinfo'},
+    "invalid_from_server_version_in_lists": {'code': "L100", 'ui_applicable': False,
```

Comment:
error codes should have 2 letter prefix
```suggestion
    "invalid_from_server_version_in_lists": {'code': "LI100", 'ui_applicable': False,
```